src/cargo/ops/cargo_package.rs

```diff
@@ -567,13 +585,14 @@ fn check_repo_state(
     }
 }
 
+/// Returns the uncompressed size of the contents of the new archive file.
```

Comment:
It's nice to add a comment. Thank you! Could we also add the purpose of this function, such as:



```suggestion
/// Packages a list of [`ArchiveFile`]s and writes into the given file.
///
/// Returns the uncompressed size of the contents of the new archive file.
```

Reply:
makes sense, done!